crates/compiler/can/src/desugar.rs

```diff
@@ -755,28 +755,6 @@ pub fn desugar_expr<'a>(
             })
         }
 
-        // Replace an empty final def with a `Task.ok {}`
```

Comment:
Hmm, does it mean that if the last line is `x!` - we await and return an empty record? So the user has to change it to `x` to return its value? If so, the whole change is not correct and I produced a breaking change 😅

At least I learned smth new and now I know where and how to fix the issue not removing that much code!

Reply:
We don't want to substitute `Task.ok {}` anywhere. `x!` should be desugared into `x` if it's the last expression